x-pack/plugins/fleet/server/routes/epm/handlers.ts

```diff
@@ -594,3 +604,27 @@ export const reauthorizeTransformsHandler: FleetRequestHandler<
     return defaultFleetErrorHandler({ error, response });
   }
 };
+
+// Don't expose the whole SO in the API response, only selected fields
+const soToInstallationInfo = (pkg: PackageListItem | PackageInfo) => {
+  if ('savedObject' in pkg && pkg.savedObject?.attributes) {
+    const { attributes } = pkg.savedObject;
+    const installationInfo: InstallationInfo = {
+      ...pick(pkg.savedObject, ['created_at', 'updated_at', 'namespaces', 'type']),
+      installed_kibana: attributes.installed_kibana,
+      installed_es: attributes.installed_es,
+      install_status: attributes.install_status,
+      install_source: attributes.install_source,
+      name: attributes.name,
+      version: attributes.version,
+      verification_status: attributes.verification_status,
+      verification_key_id: attributes.verification_key_id,
```

Comment:
`installed_kibana_space_id` and `experimental_data_stream_features` looks missing?

Reply:
You're right, strange that the linter didn't complain